diff --git a/source/Rangers.dpr b/source/Rangers.dpr
index 1948f70..5ec0990 100644
--- a/source/Rangers.dpr
+++ b/source/Rangers.dpr
@@ -4,13 +4,19 @@ program Rangers;
 {$IMAGEBASE $00400000}
 {$SETPEFLAGS $20}
 {$I-}
+{$IFNDEF CTR}
 {$R Rangers.res}
+{$ENDIF}
 
 uses
 {$IFDEF UNIX}
   cthreads,
   cwstring,
 {$ENDIF}
+{$IFDEF CTR}
+  ctrwstring,
+  GameCtrStartup,
+{$ENDIF}
 {$IFDEF MSWINDOWS}
   Windows,
 {$ENDIF}
